packages/fuel-indexer/src/executor.rs

```diff
@@ -631,12 +632,18 @@ impl WasmIndexExecutor {
             return Err(IndexerError::MissingHandler);
         }
 
-        // FunctionEnvMut and SotreMut must be scoped because they can't be used
-        // across await
-        let version = {
+        // FunctionEnvMut and SotreMut must be scoped with {} because they can't
```

Comment:
```suggestion
        // FunctionEnvMut and StoreMut must be scoped because they can't
```
The term "scoped" implies `{}`